### PR TITLE
Added example and parameter description to :class:`~.AnnularSector`

### PR DESCRIPTION
**Pull request by MathInvariance**
## Explanation for Changes

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -737,6 +737,52 @@ def __init__(self, width=2, height=1, **kwargs):
...
+    outer_radius : :class:`float`, optional
+       The outside radius of the Annular Sector.
+    angle : :class:`float`, optional
+       The clockwise angle of the Annular Sector.
+    start_angle : :class:`float`, optional
+       The starting clockwise angle of the Annular Sector.
+    fill_opacity : :class:`float`, optional
+       The opacity of the color filled in the Annular Sector.
+    stroke_width : :class:`float`, optional
+       The stroke width of the Annular Sector.
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:755]:
```suggestion
    inner_radius
       The inside radius of the Annular Sector.
    outer_radius
       The outside radius of the Annular Sector.
    angle
       The clockwise angle of the Annular Sector.
    start_angle
       The starting clockwise angle of the Annular Sector.
    fill_opacity
       The opacity of the color filled in the Annular Sector.
    stroke_width
       The stroke width of the Annular Sector.
```
The typings should be in the function parameters and not the docstrings.
See: https://docs.manim.community/en/latest/contributing/typings.html

File: manim/mobject/geometry.py [hunk 2]
```diff
@@ -737,6 +737,52 @@ def __init__(self, width=2, height=1, **kwargs):
...
+    color : optional
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:756]: This should have a typehint `Color`

File: manim/mobject/geometry.py [hunk 3]
```diff
@@ -737,6 +737,52 @@ def __init__(self, width=2, height=1, **kwargs):
...
+       The color filled into the Annular Sector.
+
+    Examples
+    --------
+    .. manim:: AnnularSectorExample
+        :save_last_frame:
+
+        class AnnularSectorExample(Scene):
+            def construct(self):
+                # Changed background color to clearly visualize changes in fill_opacity.
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:766]:
```suggestion
                # Changes background color to clearly visualize changes in fill_opacity.
```

**Review by Nikhil0504 — CHANGES_REQUESTED** [review]
Requested a few changes

File: manim/mobject/geometry.py (hunk 2 above)
Comment (Nikhil0504) [reply manim/mobject/geometry.py:756]:
```suggestion
    color
```

**Review by behackl — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**